README.md

```diff
@@ -49,6 +50,18 @@ To build all apps and packages, run the following command:
 npm run build
 ```
 
+### Running in Docker
+
+To run the application locally in a Docker container, run the following commands:
+
+```
+docker-compose build
```

Comment:
Should we mention somewhere in the readme that users need to install docker?

Reply:
That's a good point. Its an implied prereq, however we should be explicit

Reply:
@laurenrevere updated!

Reply:
Should we also note about the zscaler issue? Otherwise looks good, got it to build now!

Reply:
I'll add a quick note about it

Reply:
@nicholaslyon-dt updated!